aboutlibraries-definitions/src/main/res/values/library_cardsuiforandroid_strings.xml

```diff
@@ -5,7 +5,7 @@
     <string name="library_cardsuiForAndroid_author">nadavfirm</string>
     <string name="library_cardsuiForAndroid_authorWebsite">https://github.com/nadavfima</string>
     <string name="library_cardsuiForAndroid_libraryName">CardsUI</string>
-    <string name="library_cardsuiForAndroid_libraryDescription">Google’s Now popular Cards UI has everyone fascinated. This library aims to mimick this widget.
+    <string name="library_cardsuiForAndroid_libraryDescription">Google’s Now popular Cards UI has everyone fascinated. This library aims to mimic this widget.
```

Comment:
This library seems to be dead. Otherwise, I'd send a PR to fix its spelling. I'm happy to drop this (or any other change).